wgpu-hal/src/vulkan/adapter.rs

```diff
@@ -23,6 +23,7 @@ pub struct PhysicalDeviceFeatures {
     image_robustness: Option<vk::PhysicalDeviceImageRobustnessFeaturesEXT>,
     robustness2: Option<vk::PhysicalDeviceRobustness2FeaturesEXT>,
     depth_clip_enable: Option<vk::PhysicalDeviceDepthClipEnableFeaturesEXT>,
+    multiview: Option<vk::PhysicalDeviceMultiviewFeatures>,
```

Comment:
I think this should be `vulkan_1_1: vk::PhysicalDeviceVulkan11Features` and come right between `core` and `vulkan_1_2`.
In addition, I think we'd need `multiview: VkPhysicalDeviceMultiviewFeaturesKHR`, so that we can work with either Vk-1.1 or Vk-1.0 with extensions.